modules/gatsby/website-template/ocular-config-template.js

```diff
@@ -9,11 +9,19 @@
 
 module.exports = init => `
 module.exports = {
+  // Adjusts amount of debug information from ocular-gatsby
+  logLevel: 4,
+
   DOC_FOLDER: '${init.websiteFolder}/../docs/',
   ROOT_FOLDER: '${init.websiteFolder}/../',
 
   EXAMPLES: [
-    // {title: 'my example', path: 'examples/my-example/', image: 'images/my-example.jpg'}
+    // {
+    //   title: 'my example',
+    //   path: 'examples/my-example/',
+    //   image: 'images/my-example.jpg',
+    //   componentUrl: '../examples/app.js'
+    // }
   ],
   DOCS: [
```

Comment:
Maybe we should also provide a sample table-of-contents.json?

Reply:
Yes it could for instance just contain the top level chapters. Not sure if that would build though...